### ISSUE TITLE
Tutor Types - Overlay and Notify Accessibility Concern

### ISSUE THREAD

**Issue by tajuddinAherikar** [adaptlearning/adapt_framework#3501]
### Description:
The current setup allows Tutor types - Overlay and Notify to be used without a title. This potential scenario raises concerns regarding accessibility, as popups lacking a title may lead to accessibility issues.

### Your environment
* version (AT 0.11.2/5.33.1)
* Chrome v120.0.6099.217
* Win 11

### Steps to reproduce

1. Navigate to [specific location or screen where Tutor types - Overlay and Notify are used].
2. Trigger the Overlay and Notify functionalities without providing a title.

### Expected behaviour
Default inclusion of a title for Tutor types - Overlay and Notify to ensure accessibility standards are met.

### Actual behaviour
Tutor types - Overlay and Notify can be used without a title, potentially causing accessibility issues.

### Screenshots
**Notify**
![image](https://github.com/adaptlearning/adapt_framework/assets/49470454/80ab8c94-e4be-4bc5-b372-c4d6ca23d660)

**Overlay**
![image](https://github.com/adaptlearning/adapt_framework/assets/49470454/fa0d9f43-c1d3-4b74-bbf6-7ba2e85c1696)

**Comment by kirsty-hames** [comment]
Hi @tajuddinAherikar, thanks for raising. I recently added an [issue](https://github.com/adaptlearning/adapt-contrib-core/issues/483) for this explaining the impacts on accessibility and the core functionality/plugins this relates too.

Your issue is specific to Tutor types which should be resolved by [Core PR 484](https://github.com/adaptlearning/adapt-contrib-core/pull/484). The PR provides a default global title in the instance a `_feedback.title` isn't set. 

Does this resolve your issue?

**Comment by tajuddinAherikar** [comment]
Hi @kirsty-hames,

Thanks for your quick response. I've reviewed Core PR 484, and it seems to address the Tutor types issue by introducing a default global title when `_feedback.title` isn't set. This should resolve the problem. Appreciate it.
